### PR TITLE
Enlarge fullscreen view

### PR DESCRIPTION
**Pull request by fscholdei**
## Proposed changes

Enlarge fullscreen view

### Your checklist for this pull request

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

**Contributor**:
- [x] PR is well described and the description can be used as a commit message on squash
- [x] Related issues linked to PR if existing and labels set
- [ ] New code is covered with unit tests
- [ ] New ui components are tested inside the storybook (module ui-components only) 
- [x] [Changelog entry file](https://github.com/scm-manager/changelog#changelog-entry-files) created in `gradle/changelog`
- [ ] Feature has been tested with different permissions
- [ ] Documentation updated (only necessary for new features or changed behaviour)

**Reviewer**:
- [ ] The clean code principles are respected ([CleanCode](https://clean-code-developer.com/virtues/))
- [ ] All new code/logic is implemented on the right spot / "Should this be done here?"
- [ ] UI changes fits into the layout
- [ ] The UI views / components are responsive (mobile views)
- [ ] Correct translations are available

### Checklist for branch merge request (not required for forks)

- [ ] Branch is green/blue on [Jenkins](https://oss.cloudogu.com/jenkins/)
- [ ] Quality Gate passed on [SonarQube](https://sonarcloud.io/organizations/scm-manager/projects)

### CODE REVIEW

File: src/main/js/charts/Chart.tsx [hunk 1]
```diff
@@ -21,63 +21,70 @@
...
-      loading: true,
-      showModal: false
-    };
-  }
 
-  getStatistics() {
-    const { getData } = this.props;
+type ChartObjectProps = {
+  statisticData: StatisticData;
+  render: (props: RenderProps) => any;
```
Comment (Pilopa) [inline src/main/js/charts/Chart.tsx:56]: Please replace all instances of `any`, where possible.

**Review by Pilopa — CHANGES_REQUESTED** [review]

File: src/main/js/charts/CommitsPerAuthor.tsx [hunk 2]
```diff
@@ -21,51 +21,45 @@
...
+import React, { FC } from "react";
+import { useTranslation } from "react-i18next";
 import { Pie } from "react-chartjs-2";
 import { StatisticData } from "../DataTypes";
-import { withTranslation, WithTranslation } from "react-i18next";
 
-type Props = WithTranslation & {
+type Props = {
   statisticData: StatisticData;
   options: any;
```
Comment (fscholdei) [inline src/main/js/charts/CommitsPerAuthor.tsx:31]:
```suggestion
  options: unknown;
```

File: src/main/js/charts/Chart.tsx [hunk 3]
```diff
@@ -21,63 +21,70 @@
...
-import NoDataFound from "../NoDataFound";
 
 type RenderProps = {
   statisticData: StatisticData;
   options: any;
 };
 
-type Props = WithTranslation & {
+type Props = {
   render: (props: RenderProps) => any;
```
Comment (fscholdei) [inline src/main/js/charts/Chart.tsx:36]:
```suggestion
  render: (props: RenderProps) => unknown;
```

**Comment by Pilopa** [comment]
The type changes and conversions to functional components are getting out of hand. In my opinion, and after discussing it with the team, I would suggest to only apply the changes in the [Enlarge fullscreen view](https://github.com/scm-manager/scm-statistic-plugin/pull/37/commits/f12c104c9189da2cb37b6ddd47f3f167d8d794d5) and [Add changelog entry](https://github.com/scm-manager/scm-statistic-plugin/pull/37/commits/94a8c30e3a09e3d138679944522b54a1f5342118) commits. Everything else should be reverted, if at all possible.

**Review by Pilopa — APPROVED** [review]

**Pull request merged**